be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -52,13 +52,14 @@ SegmentIterator::~SegmentIterator() {
 
 Status SegmentIterator::_init() {
     DorisMetrics::segment_read_total.increment(1);
+    RETURN_IF_ERROR(_init_column_iterators());
```

Comment:
also update `_prepare_seek`?

Reply:
hi dayue, here just init return columns, I think there is no need to call _prepare_seek.

Reply:
OK, you're right. I didn't realize that `_init_column_iterators()` creates iterators for return columns only, maybe we should add a comment on `_init_column_iterators` or `_schema`?